test/utils/sort-users.test.js

```diff
@@ -0,0 +1,28 @@
+import { expect, describe, it } from 'vitest';
+
+
+let testArray = [
+    {familyName: 'Zabato'},
+    {familyName: 'Leonetti'},
+    {familyName: 'MacDonald'},
+    {familyName: 'Douglas'}
+]
+
+const comparePerson = (a, b) => {
+    if (a.familyName[0].toLowerCase() < b.familyName[0].toLowerCase()) {
+      return -1;
+    }
+    if (a.familyName[0].toLowerCase() > b.familyName[0].toLowerCase()) {
+      return 1;
+    }
+    return 0;
+};
+
+const results = [...testArray].sort(comparePerson)
+
+describe('Sorting', ()=> {
+    it('array should be sorted by familyName', () => {
+        testArray.sort(comparePerson)
```

Comment:
The `const` for `results` should be done within the test itself within the `it` statement since it's part of the test.

See example in `session-helper.test.js` Line 97 to Line 102.

Reply:
Also, notice how in the function used in ClientListTable, the utilizes the following structure:

```
const sortedUserList = userListCopy.sort(comparePerson);
```

For the test, we could do something similar but with `results` instead of `sortedUserList`.